lib/raygun.batch.ts

```diff
@@ -1,31 +1,33 @@
 import { sendBatch } from "./raygun.transport";
 import { startTimer } from "./timer";
 import type { IncomingMessage } from "http";
-import {
-  callVariadicCallback,
-  Callback,
-  HTTPOptions,
-  SendOptions,
-} from "./types";
+import { HTTPOptions, SendOptions } from "./types";
 
 const debug = require("debug")("raygun");
 
-export type MessageAndCallback = {
+export type MessageAndPromise = {
```

Comment:
If `MessageAndCallback` was somehow surfaced to the user, we are introducing a breaking change here

Reply:
No, it is not.